front/task.form.php

```diff
@@ -36,7 +36,7 @@
 $pfTask = new PluginGlpiinventoryTask();
 
 //Submit the task form parameters
-$pfTask->submitForm($_POST);
+$pfTask->submitForm($_GET);
```

Comment:
Other actions are still using POST, I guess using `$_REQUEST` will permit to handle all cases.

```suggestion
$pfTask->submitForm($_REQUEST);
```